<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能选择</string>
    </property>
    <addaction name="actionManage_Customers"/>
    <addaction name="actionManage_Accounts"/>
    <addaction name="actionManage_Loans"/>
    <addaction name="actionBusiness_Statistics"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionManage_Customers">
   <property name="text">
    <string>Manage Customers</string>
   </property>
  </action>
  <action name="actionssdf">
   <property name="text">
    <string>ssdf</string>
   </property>
  </action>
  <action name="actionManage_Accounts">
   <property name="text">
    <string>Manage Account</string>
   </property>
  </action>
  <action name="actionManage_Loans">
   <property name="text">
    <string>Manage Loans</string>
   </property>
  </action>
  <action name="actionBusiness_Statistics">
   <property name="text">
    <string>Business Statistics</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
